<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>232</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>232</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>232</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">xcomPose v0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Triangle meshes</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="TriangleMeshTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Actions</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonImport">
       <property name="text">
        <string>Import...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExportAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export All...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Update BIN</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShortInstruction"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="flags.qrc">
      <normaloff>:/flags/gb.png</normaloff>:/flags/gb.png</iconset>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionDeutsch"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionExportAll"/>
    <addaction name="separator"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionSetBinPath"/>
    <addaction name="actionUpdateBin"/>
    <addaction name="actionCompleteBinUpdate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
   <addaction name="menuLanguage"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About xcomPose...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="flags.qrc">
     <normaloff>:/flags/gb.png</normaloff>:/flags/gb.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionDeutsch">
   <property name="icon">
    <iconset resource="flags.qrc">
     <normaloff>:/flags/de.png</normaloff>:/flags/de.png</iconset>
   </property>
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="iconText">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionExportAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export All...</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionSetBinPath">
   <property name="text">
    <string>Set BIN Path...</string>
   </property>
  </action>
  <action name="actionUpdateBin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Update BIN</string>
   </property>
  </action>
  <action name="actionCompleteBinUpdate">
   <property name="text">
    <string>Rebuild BIN</string>
   </property>
  </action>
  <action name="actionShortInstruction">
   <property name="text">
    <string>Short Instruction...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TriangleMeshTabWidget</class>
   <extends>QTabWidget</extends>
   <header>trianglemeshtabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="flags.qrc"/>
 </resources>
 <connections/>
</ui>
